spec/factories/orders.rb

```diff
@@ -69,11 +68,12 @@
       origin_volume { volume.to_d }
       locked { price.to_d *  volume.to_d }
       origin_locked { locked.to_d }
-      member { create(:member) }
     end
   end
 
   factory :order_ask do
+    member { create(:member) }
```

Comment:
```suggestion
    member
```
и просто такой вариант вроде как должен подойти